nengo_gui/templates/page.html

```diff
@@ -94,6 +94,9 @@
                 <li id='Redo_last_button' role="presentation">
                     <a title='Redo last' class='glyphicon glyphicon-share-alt'> </a>
                 </li>
+                <li id='Forum_link' role="presentation" onclick="window.open('http://forum.nengo.ai');">
```

Comment:
The link should be https not http (it'll redirect automatically, but might as well put in the right URL)